### PR TITLE
adding pkce link, em dash clean up, sample code na removed, other nits

### PR DESCRIPTION
**Pull request by susanharper-okta**
## Description:
- **What's changed?** adding pkce link, em dash clean up, sample code na removed, other nits 
- **Is this PR related to a Monolith release?** No

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/index.md [hunk 1]
```diff
@@ -63,27 +59,27 @@ To connect your org to the Identity Provider, add and configure that Identity Pr
 
 You can test your integration by configuring a [routing rule](https://help.okta.com/okta_help.htm?id=ext-cfg-routing-rules) to use <StackSelector snippet="idp" noSelector inline /> as the Identity Provider.
 
-### Use the Authorize URL to simulate the authorization flow
-
-You can also use the Authorize URL to simulate the authorization flow. The Okta Identity Provider that you created generated an authorize URL with a number of blank parameters that you can fill in to test the flow with the Identity Provider. The authorize URL initiates the authorization flow that authenticates the user with the Identity Provider.
+Alternatively, you can use the Authorize URL to simulate the authorization flow. The Okta Identity Provider that you created generated an authorize URL with a number of blank parameters that you can fill in to test the flow with the Identity Provider. The authorize URL initiates the authorization flow that authenticates the user with the Identity Provider.
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/index.md:62]:
```suggestion
Alternatively, you can use the Authorize URL to simulate the authorization flow. The Okta Identity Provider that you created generates an authorize URL with a number of blank parameters that you can fill in to test the flow with the Identity Provider. The authorize URL initiates the authorization flow that authenticates the user with the Identity Provider.
```

File: packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/index.md [hunk 2]
```diff
@@ -63,27 +59,27 @@ To connect your org to the Identity Provider, add and configure that Identity Pr
...
 
-> **Note:** Use this step to test your authorization URL as an HTML link. For information on using the Sign-In Widget, Okta-hosted sign-in page, or AuthJS, see [Use the Identity Provider to sign in](/docs/guides/add-an-external-idp/apple/main/#use-the-identity-provider-to-sign-in).
+> **Note:** Use this step to test your authorization URL as an HTML link. For information on testing your authorization URL using the Sign-In Widget, Okta-hosted sign-in page, or AuthJS, see the [next section](/docs/guides/add-an-external-idp/-/main/#use-the-identity-provider-to-sign-in).
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/index.md:64]:
```suggestion
> **Note:** Use this step to test your authorization URL as an HTML link. For information on testing your authorization URL using the Sign-In Widget, Okta-hosted sign-in page, or AuthJS, see the [next section](#use-the-identity-provider-to-sign-in).
```
I'm not sure if you need the full path? If the full path is required for the structure of these docs, then just ignore this suggestion.

File: packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/index.md [hunk 3]
```diff
@@ -63,27 +59,27 @@ To connect your org to the Identity Provider, add and configure that Identity Pr
...
+>
+> If you are using Authorization Code with PKCE as the grant type, you must generate and store the PKCE. See [Implement authorization by grant type](https://developer.okta.com/docs/guides/implement-grant-type/authcodepkce/main/#flow-specifics). When you use this grant type, we recommend that you use the [AuthJS SDK](https://github.com/okta/okta-auth-js#signinwithredirectoptions).
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/index.md:66]:
```suggestion
> If you are using Authorization Code with PKCE as the grant type, you must generate and store the PKCE. See [Implement authorization by grant type](/docs/guides/implement-grant-type/authcodepkce/main/#flow-specifics). Okta recommends that you use the [AuthJS SDK](https://github.com/okta/okta-auth-js#signinwithredirectoptions) with this grant type.
```

File: packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/index.md [hunk 4]
```diff
@@ -63,27 +59,27 @@ To connect your org to the Identity Provider, add and configure that Identity Pr
...
 
 In the URL, replace `${yourOktaDomain}` with your org's base URL, and then replace the following values:
 
-* `client_id` &mdash; use the `client_id` value that you obtained from the OpenID Connect client application in the previous section. This is not the `client_id` from the Identity Provider.
+* `client_id`: Use the `client_id` value that you obtained from the OpenID Connect client application in the previous section. This is not the `client_id` from the Identity Provider.
 
-* `response_type` &mdash; determines which flow is used. For the [Implicit](/docs/guides/implement-grant-type/implicit/main/) flow, this should be `id_token`. For the [Authorization Code](/docs/guides/implement-grant-type/authcode/main/) flow, this should be `code`.
+* `response_type`: Determines which flow is used. For the [Implicit](/docs/guides/implement-grant-type/implicit/main/) flow, this should be `id_token`. For the [Authorization Code](/docs/guides/implement-grant-type/authcode/main/) flow, this should be `code`.
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/index.md:72]:
```suggestion
* `response_type`: Determines which flow is used. For the [Implicit](/docs/guides/implement-grant-type/implicit/main/) flow, use `id_token`. For the [Authorization Code](/docs/guides/implement-grant-type/authcode/main/) flow, use `code`.
```

File: packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/index.md [hunk 5]
```diff
@@ -63,27 +59,27 @@ To connect your org to the Identity Provider, add and configure that Identity Pr
...
 
-* `response_mode` &mdash; determines how the authorization response should be returned. This should be `fragment`.
+* `response_mode`: Determines how the authorization response should be returned. This should be `fragment`.
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/index.md:74]:
```suggestion
* `response_mode`: Determines how the authorization response is returned. Use `fragment`.
```

File: packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/index.md [hunk 6]
```diff
@@ -63,27 +59,27 @@ To connect your org to the Identity Provider, add and configure that Identity Pr
...
 
-* `scope` &mdash; determines the claims that are returned in the ID token. Include the scopes that you want to request authorization for and separate each by a space. You need to include at least the `openid` scope. You can request any of the standard OpenID Connect scopes about users, such as `profile` and `email` as well as any custom scopes specific to your Identity Provider.
+* `scope`: Determines the claims that are returned in the ID token. Include the scopes that you want to request authorization for and separate each by a space. You need to include at least the `openid` scope. You can request any of the standard OpenID Connect scopes about users, such as `profile` and `email` as well as any custom scopes specific to your Identity Provider.
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/index.md:76]:
```suggestion
* `scope`: Determines the claims that are returned in the ID token. Include the scopes that you want to request authorization for and separate each by a space. You need to include at least the `openid` scope. You can request any of the standard OpenID Connect scopes about users, such as `profile` and `email`, as well as any custom scopes specific to your Identity Provider.
```

File: packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/oktatookta/appidpinokta.md [hunk 7]
```diff
@@ -1,18 +1,18 @@
...
 
-* **Issuer** &mdash; the identifier of the Okta Identity Provider. For example, the Okta org where you created the Identity Provider app: `https://${theOktaIdPOrg}`<br>
-* **Authorization endpoint** &mdash; the URL of the Okta Identity Provider's OAuth 2.0 authorization endpoint. For example: `https://${theOktaIdPOrg}/oauth2/v1/authorize`
-* **Token endpoint** &mdash; the URL of the Okta Identity Provider's token endpoint for obtaining access and ID tokens. For example: `https://${theOktaIdPOrg}/oauth2/v1/token`
-* **JWKS endpoint** &mdash; the URL of the Okta Identity Provider's JSON Web Key Set document. This document contains signing keys that are used to validate the signatures from the provider. For example: `https://${theOktaIdPOrg}/oauth2/v1/keys`
-* **Userinfo endpoint (optional)** &mdash; the endpoint for getting identity information about the user. For example: `https://${theOktaIdPOrg}/oauth2/v1/userinfo`
+* **Issuer**: The identifier of the Okta Identity Provider. For example, the Okta org where you created the Identity Provider app: `https://${theOktaIdPOrg}`<br>
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/oktatookta/appidpinokta.md:12]: is <br> required here?

File: packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/openidconnect/appidpinokta.md [hunk 8]
```diff
@@ -1,18 +1,18 @@
...
 
-* **Issuer** &mdash; the identifier of the <StackSelector snippet="idp" noSelector inline /> provider. For example, where you created the Identity Provider app: `https://${theIdPdomain}/`
-* **Authorization endpoint** &mdash; the URL of the Identity Provider's OAuth 2.0 authorization endpoint. For example: `https://${theIdPdomain}/oauth2/v1/authorize`
-* **Token endpoint** &mdash; the URL of the Identity Provider's token endpoint for obtaining access and ID tokens. For example: `https://${theIdPdomain}/oauth2/v1/token`
-* **JWKS endpoint** &mdash; the URL of the Identity Provider's JSON Web Key Set document. This document contains signing keys that are used to validate the signatures from the provider. For example: `https://${theIdPdomain}/oauth2/v1/keys`
-* **Userinfo endpoint (optional)** &mdash; the endpoint for getting identity information about the user. For example: `https://${theIdPdomain}/oauth2/v1/userinfo`.
+* **Issuer**: The identifier of the <StackSelector snippet="idp" noSelector inline /> provider. For example, where you created the Identity Provider app: `https://${theIdPdomain}/`
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/openidconnect/appidpinokta.md:12]:
```suggestion
* **Issuer**: The identifier of the <StackSnippet snippet="idp" inline /> provider. For example, `https://${theIdPdomain}/`.
```
where you created the Identity Provider app?

File: packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/saml2/appatidp.md [hunk 9]
```diff
@@ -4,6 +4,6 @@ Access the external <StackSelector snippet="idp" noSelector inline /> Identity P
...
 
-* **IdP Issuer URI** &mdash; the issuer URI of the Identity Provider. This value is usually the <StackSelector snippet="idp" noSelector inline /> Metadata `entityID` of the Identity Provider `EntityDescriptor`.
-* **IdP Single Sign-On URL** &mdash; the binding specific Identity Provider Authentication Request Protocol endpoint that receives <StackSelector snippet="idp" noSelector inline /> AuthN Request messages from Okta.
-* **IdP Signature Certificate** &mdash; the PEM or DER encoded public key certificate of the Identity Provider used to verify <StackSelector snippet="idp" noSelector inline /> message and assertion signatures.
+* **IdP Issuer URI**: The issuer URI of the Identity Provider. This value is usually the <StackSelector snippet="idp" noSelector inline /> Metadata `entityID` of the Identity Provider `EntityDescriptor`.
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/saml2/appatidp.md:7]:
```suggestion
* **IdP Issuer URI**: The issuer URI of the Identity Provider. This value is usually the <StackSnippet snippet="idp" inline /> Metadata `entityID` of the Identity Provider `EntityDescriptor`.
```

File: packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/saml2/appatidp.md [hunk 10]
```diff
@@ -4,6 +4,6 @@ Access the external <StackSelector snippet="idp" noSelector inline /> Identity P
...
+* **IdP Single Sign-On URL**: The binding specific Identity Provider Authentication Request Protocol endpoint that receives <StackSelector snippet="idp" noSelector inline /> AuthN Request messages from Okta.
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/saml2/appatidp.md:8]:
```suggestion
* **IdP Single Sign-On URL**: The binding specific Identity Provider Authentication Request Protocol endpoint that receives <StackSnippet snippet="idp" inline /> AuthN Request messages from Okta.
```

File: packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/saml2/appatidp.md [hunk 11]
```diff
@@ -4,6 +4,6 @@ Access the external <StackSelector snippet="idp" noSelector inline /> Identity P
...
+* **IdP Signature Certificate**: The PEM or DER encoded public key certificate of the Identity Provider used to verify <StackSelector snippet="idp" noSelector inline /> message and assertion signatures.
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/saml2/appatidp.md:9]:
```suggestion
* **IdP Signature Certificate**: The PEM or DER encoded public key certificate of the Identity Provider used to verify <StackSnippet snippet="idp" inline /> message and assertion signatures.
```

File: packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/saml2/appidpinokta.md [hunk 12]
```diff
@@ -1,8 +1,8 @@
-* **Name** &mdash; enter the name that you would expect to see on a button, such as **Sign in with <StackSelector snippet="idp" noSelector inline /> 2.0**.
+* **Name**: Enter the name that you would expect to see on a button, such as **Sign in with <StackSelector snippet="idp" noSelector inline /> 2.0**.
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/saml2/appidpinokta.md:1]:
```suggestion
* **Name**: Enter the name that you would expect to see on a button, such as **Sign in with <StackSnippet snippet="idp" inline /> 2.0**.
```

File: packages/@okta/vuepress-site/docs/guides/social-login/main/index.md [hunk 13]
```diff
@@ -55,10 +51,10 @@ To add <StackSelector snippet="idp" noSelector inline /> as an Identity Provider
 1. Click **Next**.
 1. In the **General Settings** section, define the following:
 
-    * **Name** &mdash; Enter a name for the Identity Provider in Okta.
-    * **Client ID** &mdash; Paste the generated client ID from your <StackSelector snippet="idp" noSelector inline /> application.
-    * **Client Secret** &mdash; Paste the generated client secret from your <StackSelector snippet="idp" noSelector inline /> application.
-    * **Scopes** &mdash; Leave the defaults for a simple sign-in flow. You can also add more scopes. See <StackSelector snippet="scopes" noSelector inline />.
+    * **Name**: Enter a name for the Identity Provider in Okta.
+    * **Client ID**: Paste the generated client ID from your <StackSelector snippet="idp" noSelector inline /> application.
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/social-login/main/index.md:55]:
```suggestion
    * **Client ID**: Paste the generated client ID from your <StackSnippet snippet="idp" inline /> application.
```

File: packages/@okta/vuepress-site/docs/guides/social-login/main/index.md [hunk 14]
```diff
@@ -55,10 +51,10 @@ To add <StackSelector snippet="idp" noSelector inline /> as an Identity Provider
...
+    * **Client Secret**: Paste the generated client secret from your <StackSelector snippet="idp" noSelector inline /> application.
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/social-login/main/index.md:56]:
```suggestion
    * **Client Secret**: Paste the generated client secret from your <StackSnippet snippet="idp" inline /> application.
```

File: packages/@okta/vuepress-site/docs/guides/social-login/main/index.md [hunk 15]
```diff
@@ -55,10 +51,10 @@ To add <StackSelector snippet="idp" noSelector inline /> as an Identity Provider
...
+    * **Scopes**: Leave the defaults for a simple sign-in flow. You can also add more scopes. See <StackSelector snippet="scopes" noSelector inline />.
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/social-login/main/index.md:57]:
```suggestion
    * **Scopes**: Leave the defaults for a simple sign-in flow. You can also add more scopes. See <StackSnippet snippet="scopes" inline />.
```

File: packages/@okta/vuepress-site/docs/guides/social-login/main/index.md [hunk 16]
```diff
@@ -70,19 +66,19 @@ Alternatively, you can use the Authorize URL to simulate the authorization flow.
 
 In the URL, replace `${yourOktaDomain}` with your org's base URL, and then replace the following values:
 
-* `client_id` &mdash; use the `client_id` value from your Okta app integration. This is not the `client_id` from the Identity Provider. For example, `0oawjqpb2wcUAWM8C0h7`.
+* `client_id`: Use the `client_id` value from your Okta app integration. This is not the `client_id` from the Identity Provider. For example, `0oawjqpb2wcUAWM8C0h7`.
 
-* `response_type` &mdash; determines which flow is used. For the [Implicit](/docs/guides/implement-grant-type/implicit/main/) flow, this should be `id_token`. For the [Authorization Code](/docs/guides/implement-grant-type/authcode/main/) flow, this should be `code`.
+* `response_type`: Determines which flow is used. For the [Implicit](/docs/guides/implement-grant-type/implicit/main/) flow, this should be `id_token`. For the [Authorization Code](/docs/guides/implement-grant-type/authcode/main/) flow, this should be `code`.
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/social-login/main/index.md:71]:
```suggestion
* `response_type`: Determines which flow is used. For the [Implicit](/docs/guides/implement-grant-type/implicit/main/) flow, use `id_token`. For the [Authorization Code](/docs/guides/implement-grant-type/authcode/main/) flow, use `code`.
```

File: packages/@okta/vuepress-site/docs/guides/social-login/main/index.md [hunk 17]
```diff
@@ -70,19 +66,19 @@ Alternatively, you can use the Authorize URL to simulate the authorization flow.
...
 
-* `response_mode` &mdash; determines how the authorization response should be returned. This should be `fragment`.
+* `response_mode`: Determines how the authorization response should be returned. This should be `fragment`.
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/social-login/main/index.md:73]:
```suggestion
* `response_mode`: Determines how the authorization response is returned. Use `fragment`.
```

File: packages/@okta/vuepress-site/docs/guides/social-login/main/index.md [hunk 18]
```diff
@@ -94,10 +90,9 @@ https://${yourOktaDomain}/oauth2/v1/authorize?idp=${idp_id}&client_id=${client_i
...
+## Add <StackSnippet snippet="idp" inline /> to the Okta Sign-In Widget
 
-The [Okta Sign-In Widget](https://github.com/okta/okta-signin-widget) is an embeddable JavaScript Widget that reproduces the look and behavior of the standard Okta sign-in page. You can add a **Sign in with <StackSelector snippet="idp" noSelector inline />** button to the Widget by adding the following code to your Okta Sign-In Widget configuration. Replace `Your_IDP_ID` with the Identity Provider ID from your Identity Provider that you created in Okta in the 
-[Create the Identity Provider in Okta](#create-the-identity-provider-in-okta) section.
+The [Okta Sign-In Widget](https://github.com/okta/okta-signin-widget) is an embeddable JavaScript Widget that reproduces the look and behavior of the standard Okta sign-in page. You can add a **Sign in with <StackSelector snippet="idp" noSelector inline />** button to the Widget by adding the following code to your Okta Sign-In Widget configuration. Replace `Your_IDP_ID` with the Identity Provider ID from your Identity Provider that you created in Okta in the [Create the Identity Provider in Okta](#create-the-identity-provider-in-okta) section.
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/social-login/main/index.md:95]:
```suggestion
The [Okta Sign-In Widget](https://github.com/okta/okta-signin-widget) is an embeddable JavaScript Widget that reproduces the look and behavior of the standard Okta sign-in page. You can add a **Sign in with <StackSnippet snippet="idp" inline />** button to the Widget by adding the following code to your Okta Sign-In Widget configuration. Replace `Your_IDP_ID` with the Identity Provider ID from your Identity Provider that you created in Okta in the [Create the Identity Provider in Okta](#create-the-identity-provider-in-okta) section.
```

**Review by vanngo-okta — CHANGES_REQUESTED** [review]
Phew! Thanks for updating these docs! The new file structure took a while to get used to.

File: packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/apple/appatidp.md [hunk 19]
```diff
@@ -2,9 +2,9 @@
 
 There are three steps necessary to configure the Identity Provider at <StackSelector snippet="idp" noSelector inline />:
```
[the remark this answers is not in the document]
Comment (susanharper-okta) [reply packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/apple/appatidp.md:3]: @vanngo-okta - I don't think so. This an intro of the three steps that are required to configure the IdP..the doc then goes on to have these as doc sections and steps within the doc. I wouldn't number these.

Create an App ID
Create a Services ID 
Create a private key for client authentication

File: packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/oktatookta/appidpinokta.md (hunk 7 above)
Comment (susanharper-okta) [reply packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/oktatookta/appidpinokta.md:12]: @vanngo-okta I don't understand what you are asking

Comment (vanngo-okta) [reply packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/oktatookta/appidpinokta.md:12]: lol, it didn't render.. is the `<br>` required after this line?

Comment (susanharper-okta) [reply packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/oktatookta/appidpinokta.md:12]: oh probably not - probably left over from something else.I'll remove it!

**Review by vanngo-okta — APPROVED** [review]
LGTM! 🚀

**Comment by susanharper-okta** [comment]
> LGTM! 🚀

I didn't even LOOK in all of those files that I was fixing the em dash in to fix the Stack Selector tag. Thank you!

**Pull request merged**